fluidBuild.config.cjs

```diff
@@ -45,6 +45,7 @@ module.exports = {
 		"build:copy": [],
 		"build:genver": [],
 		"typetests:gen": ["^tsc", "build:genver"], // we may reexport type from dependent packages, needs to build them first.
+		"ts2esm": [],
 		"tsc": tscDependsOn,
 		"build:esnext": [...tscDependsOn, "^build:esnext"],
 		"build:test": [
```

Comment:
@tylerbutler, There is a comment that `build:test` doesn't need ...tscDependsOn. However, it does need `ts2esm`. Should `ts2esm` be an explicit dependency?

Reply:
It's not technically needed since it depends on tsc, which depends on everything in tscDependsOn. So the comment is  still accurate but a little vague. I'll add these details to the comment.